content/services/support/what_can_i_expect.rst

```diff
@@ -54,3 +58,6 @@ You can get this type of support with a `Success Pack <https://www.odoo.com/pric
 With a pack, one of our consultants will analyze the way your business runs and
 tell you how you can get the most out of your Odoo Database. We will handle all
 configurations and coach you on how to use Odoo.
+
+.. seealso::
+   :doc:`../../administration/upgrade`
```

Comment:
Same here.
`doc` tags are ok, however `refs` are better since they're more resilient against broken links 😉 
```suggestion
   :doc:`/administration/upgrade`
```